install/kubernetes/helm/istio/charts/security/templates/deployment.yaml

```diff
@@ -28,9 +28,27 @@ spec:
             - --append-dns-names=true
             - --grpc-port=8060
             - --grpc-hostname=citadel
-            - --self-signed-ca=true
             - --citadel-storage-namespace={{ .Release.Namespace }}
+          {{- if .Values.global.multicluster.enabled }}
```

Comment:
Can you use a separate setting ( 'security.selfSigned' or similar ) ? This is not specific to multicluster.